### PR TITLE
Remove createClass usage (from @IcanDivideBy0) and getDefaultProps usage

### PR DESCRIPTION
**Pull request by bkonkle**
Both issues cause deprecation warnings in the latest React. Thanks!

### CODE REVIEW

File: src/index.js
Comment (IcanDivideBy0) [inline src/index.js]: I guess we'll get static properties in language before the PR get merged xD

**Review by IcanDivideBy0 — APPROVED** [review]

**Comment by mxstbr** [comment]
@acdlite just gave me commit access, going to clean this up a bit. Thanks for contributing!

**Pull request merged**
